pkg/security/podsecuritypolicy/util/util.go

```diff
@@ -40,102 +41,102 @@ func GetAllFSTypesExcept(exceptions ...string) sets.String {
 func GetAllFSTypesAsSet() sets.String {
 	fstypes := sets.NewString()
 	fstypes.Insert(
-		string(policy.HostPath),
-		string(policy.AzureFile),
-		string(policy.Flocker),
-		string(policy.FlexVolume),
-		string(policy.EmptyDir),
-		string(policy.GCEPersistentDisk),
-		string(policy.AWSElasticBlockStore),
-		string(policy.GitRepo),
-		string(policy.Secret),
-		string(policy.NFS),
-		string(policy.ISCSI),
-		string(policy.Glusterfs),
-		string(policy.PersistentVolumeClaim),
-		string(policy.RBD),
-		string(policy.Cinder),
-		string(policy.CephFS),
-		string(policy.DownwardAPI),
-		string(policy.FC),
-		string(policy.ConfigMap),
-		string(policy.VsphereVolume),
-		string(policy.Quobyte),
-		string(policy.AzureDisk),
-		string(policy.PhotonPersistentDisk),
-		string(policy.StorageOS),
-		string(policy.Projected),
-		string(policy.PortworxVolume),
-		string(policy.ScaleIO),
-		string(policy.CSI),
+		string(policyv1beta1.HostPath),
+		string(policyv1beta1.AzureFile),
+		string(policyv1beta1.Flocker),
+		string(policyv1beta1.FlexVolume),
+		string(policyv1beta1.EmptyDir),
+		string(policyv1beta1.GCEPersistentDisk),
+		string(policyv1beta1.AWSElasticBlockStore),
+		string(policyv1beta1.GitRepo),
+		string(policyv1beta1.Secret),
+		string(policyv1beta1.NFS),
+		string(policyv1beta1.ISCSI),
+		string(policyv1beta1.Glusterfs),
+		string(policyv1beta1.PersistentVolumeClaim),
+		string(policyv1beta1.RBD),
+		string(policyv1beta1.Cinder),
+		string(policyv1beta1.CephFS),
+		string(policyv1beta1.DownwardAPI),
+		string(policyv1beta1.FC),
+		string(policyv1beta1.ConfigMap),
+		//string(policyv1beta1.VsphereVolume),
```

Comment:
obviously internal psp has some definitions which the external doesn't have. can we straightly copy these missing ones instead of commenting like this? 

@php-coder is it intended? i think it should have been done in  #61608 #54933.

Reply:
The commenting here is just a workaround for test failures. :slightly_smiling_face: 

I'm not sure what to do with those, so I would appreciate any ideas. I think copying those to external could work, but I would like to get some approval first.

Reply:
As far as I see the list of volumes types is just incomplete. Please create a commit filling it up with everything that is missing.